core/models.py

```diff
@@ -119,6 +120,7 @@ class PacuSession(Base, ModelUpdateMixin):
     Inspector = Column(JSONType, nullable=False, default=dict)
     Lambda = Column(JSONType, nullable=False, default=dict)
     Lightsail = Column(JSONType, nullable=False, default=dict)
+    RDS = Column(JSONType, nullable=False, default=dict)
```

Comment:
This is done correctly. Updating the models is a breaking change. This will need to be taken into consideration when merging as it will invalidate all existing db files.